internal/signing/channel_account_db_signature_client_test.go

```diff
@@ -0,0 +1,154 @@
+package signing
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stellar/go/keypair"
+	"github.com/stellar/go/network"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/wallet-backend/internal/signing/channelaccounts"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestChannelAccountDBSignatureClientGetAccountPublicKey(t *testing.T) {
+	t.Parallel()
```

Comment:
Why are using `t.Parallel()` here?

Reply:
Because the function tested has a sleep statement, we will block the other tests for at least 6 seconds if we don't run it in parallel.